rest/route/host.go

```diff
@@ -312,3 +312,64 @@ func getLimit(vals url.Values) (int, error) {
 
 	return limit, nil
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// GET /host/range
```

Comment:
I'm not sure what "range" means? couldn't this be filter?

Reply:
done